inc/class-comment-form-hacks.php

```diff
@@ -1,16 +1,14 @@
 <?php
-/**
- * Add comment note.
- *
- * @package YoastCommentHacks
- */
+
+namespace Yoast\WP\Comment\Inc;
 
 /**
- * Class YoastCommentFormHacks.
+ * Add comment note.
  *
  * @since 1.3
+ * @since 1.6.0 Class renamed from `YoastCommentFormHacks` to `Yoast\WP\Comment\Inc\Forms`.
  */
-class YoastCommentFormHacks {
+class Yoast_Comment_Forms {
```

Comment:
This should be just `Forms`, right?

Reply:
@johannadevos Cricky! Not sure how that one slipped past me. Thank you for catching this. Fixed now.